cms-adapter/src/Rsbc.Dmf.CaseManagement.Service/Protos/cmsAdapter.proto

```diff
@@ -131,172 +134,173 @@ service CaseManager {
+  string UserId = 4;
+  string CaseId = 5;
+  string CommentId = 6;
+  google.protobuf.Timestamp CommentDate = 7;
+  Driver Driver = 8;
+  string Assignee = 9;
+  string SignatureName = 10;
 }
 
-message GetDocumentRequest {	
-    string documentId = 1;
+message GetDocumentRequest {
+  string documentId = 1;
 }
 
-message GetLegacyDocumentReply {	
-    ResultStatus resultStatus = 1;
-    string errorDetail = 2;
-    LegacyDocument document = 3;
+message GetLegacyDocumentReply {
+  ResultStatus resultStatus = 1;
+  string errorDetail = 2;
+  LegacyDocument document = 3;
 }
 
-message PdfDocumentReply {	
-    ResultStatus resultStatus = 1;
-    string errorDetail = 2;
-    repeated PdfDocument pdfDocuments = 3;
+message PdfDocumentReply {
+  ResultStatus resultStatus = 1;
+  string errorDetail = 2;
+  repeated PdfDocument pdfDocuments = 3;
 }
 
 message PdfDocument{
   string pdfDocumentId = 1;
   enum StatusCodeOptions{
-      SendToBCMail = 0;
-      Sent = 1;
-      FailedToSend = 2;
-   }
+    SendToBCMail = 0;
+    Sent = 1;
+    FailedToSend = 2;
+  }
   StatusCodeOptions statusCode = 2;
   string filename = 3;
   string serverUrl = 4;
 }
 
 message CaseDetail {
-    string caseId = 1;
-    string title = 2;
-    string idCode = 3;
-    google.protobuf.Timestamp openedDate = 4;
-    string caseType = 5;
-    string dmerType = 6;
-    string status = 7;
-    string assigneeTitle = 8;
-    google.protobuf.Timestamp lastActivityDate = 9;
-    string latestDecision = 10;
-    string decisionForClass = 11;
-    google.protobuf.Timestamp decisionDate = 12;
-    google.protobuf.Timestamp dpsProcessingDate = 13;
-    int64 caseSequence = 14;
-    string driverId = 15;
-    int64 outstandingDocuments = 16;
-    string eligibleLicenseClass = 17;
-    string name = 18;
-    string driverLicenseNumber = 19;
-    google.protobuf.Timestamp birthDate = 20;
-    string firstName = 21;
-    string lastName = 22;
-    string middlename = 23;
-    google.protobuf.Timestamp latestComplianceDate = 24;
-}
-
-message CreateStatusReply {	
-    ResultStatus resultStatus = 1;
-    string errorDetail = 2;
-    string id = 3;
+  string caseId = 1;
+  string title = 2;
+  string idCode = 3;
+  google.protobuf.Timestamp openedDate = 4;
+  string caseType = 5;
+  string dmerType = 6;
+  string status = 7;
+  string assigneeTitle = 8;
+  google.protobuf.Timestamp lastActivityDate = 9;
+  string latestDecision = 10;
+  string decisionForClass = 11;
+  google.protobuf.Timestamp decisionDate = 12;
+  google.protobuf.Timestamp dpsProcessingDate = 13;
+  int64 caseSequence = 14;
+  string driverId = 15;
+  int64 outstandingDocuments = 16;
+  string eligibleLicenseClass = 17;
+  string name = 18;
+  string driverLicenseNumber = 19;
+  google.protobuf.Timestamp birthDate = 20;
+  string firstName = 21;
+  string lastName = 22;
+  string middlename = 23;
+  google.protobuf.Timestamp latestComplianceDate = 24;
+  repeated MedicalConditionItem MedicalConditions = 25;
```

Comment:
note to self: test all services that touch CaseDetails

Reply:
I'm currently leaning towards not needing this change and use "GetAllMedicalConditions" service instead